docs/reference/ontology-landscape.md

```diff
@@ -0,0 +1,561 @@
+## The Landscape of Disease and Phenotype Ontologies
```

Comment:
rename file or do you intende this to be about all kinds of ontologies? maybe better: `medical-ontology-landscape.md`?

Reply:
I don't intend to make this about all kinds of ontologies but someone else could. :) I'll rename to medical-ontology-landscape